src/Models/Auth0User.php

```diff
@@ -17,6 +17,16 @@ public function getAccountNo()
         return $this->company['navision_account_no'] ?? null;
     }
 
+    /**
+     * Return the Can book with booking numbers permission
+     *
+     * @return boolean|null
+     */
+    public function canBookWithBookingNumber(): ?bool
```

Comment:
Shouldn't this always return a boolean if `can_book_with_booking_numbers` is a bool or null?